tests/cycles.fs

```diff
@@ -11,6 +11,7 @@ testing cycle counts
 \ Take care when editing this file as the whitespace on the ends of lines is
 \ desired to keep the CYCLE: counts lined up.
 
+20 nc-limit !  \ restore default that tali.fs messes with
```

Comment:
Thanks for catching this.  Most of the cycle tests are directly testing built-in words, so it probably didn't matter much before we starting compiling words and measuring the compilation.  

Does it make more sense to put this into tali.fs at the end?